frontstage/templates/surveys/surveys.html

```diff
@@ -11,7 +11,9 @@
         </div>
 
         <div id="SUBMIT_BY_COLUMN_HEADING" class="grid__col col-2@m">
+            {% if not history %}
             Submit by
```

Comment:
I'd probably say to indent this, but that's preference.

Reply:
Not sure what the standard is here, I usually don't indent on the if/for statements so the html stays properly indented